.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
The editor config can be only locally excluded by using `.git/info/exclude` instead of `.gitignore` (same syntax).